cli/flox/src/commands/mod.rs

```diff
@@ -1807,4 +1841,26 @@ mod tests {
 
         assert!(result.unwrap().is_none());
     }
+
+    // test that update_instructions provides default message when update-instructions.txt file
+    // does not exits
+    #[tokio::test]
```

Comment:
I don't think either of these tests actually do anything async, so you can remove the `async` and swap out `#[tokio::test]` for `#[test]`.

Reply:
ah right... too much copy paste